### DISCUSSION TITLE
Welcome to LoginPage_PHP-MySql Discussions!

### DISCUSSION THREAD

**Discussion by Alessandro-DiGiusto** [Alessandro-DiGiusto/CRM-FullCode-NO-framework#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dexon-Montreal** [comment]
Hi! I saw your work! it is very well done and neat, I like the style you used I will use it for my project, I hope I can do it (?)
One last thing, the change of pages happens through links to different .php files for each page, was it a choice or a constraint?

![STolLU9B_400x400](https://user-images.githubusercontent.com/116878743/198555744-c509a325-e7b4-4e49-bec1-623bb154cb6b.jpg)

**Reply by Alessandro-DiGiusto** [reply]
1.I am satisfied that you like my work, in my free time I delight here on this project on github in the design of this CRM as a hobby, or better I would say, as a challenge, as a challenge in the sense of being able to achieve all this deliberately without the help of framework.
2.Of course you can use my code, absolutely. I grant you.

3. It was a deliberate choice.

**Reply by Dexon-Montreal** [reply]
Thanks man!
